integration-tests/suites/http_endpoint_availability.go

```diff
@@ -0,0 +1,47 @@
+package suites
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+)
+
+type HttpEndpointAvailabilityTestSuite struct {
+	IntegrationTestSuiteBase
+	Port             int
+	CollectorOptions collector.StartupOptions
+	Endpoints        []string
+}
+
+func (s *HttpEndpointAvailabilityTestSuite) SetupSuite() {
+	s.StartCollector(false, &s.CollectorOptions)
+}
+
+func (s *HttpEndpointAvailabilityTestSuite) TestAvailability() {
+	for _, endpoint := range s.Endpoints {
+		url := fmt.Sprintf("http://localhost:%d%s", s.Port, endpoint)
```

Comment:
At some point we might want to add a way to get the IP collector can be found to the `collector.Manager` interface, or even a method for getting the full URL to the introspection port, for now this is fine.